internal/testing/testdata/testdata.go

```diff
@@ -1033,27 +1033,17 @@ var (
 			DepPkg:          cdxResteasyPack,
 			DepPkgMatchFlag: model.MatchFlags{Pkg: model.PkgMatchTypeSpecificVersion},
 			IsDependency: &model.IsDependencyInputSpec{
-				DependencyType: model.DependencyTypeUnknown,
-				VersionRange:   "2.13.4.Final",
-				Justification:  isDepJustifyTopPkgJustification,
-			},
-		},
-		{
-			Pkg:             cdxTopQuarkusPack,
-			DepPkg:          cdxReactiveCommonPack,
-			DepPkgMatchFlag: model.MatchFlags{Pkg: model.PkgMatchTypeSpecificVersion},
-			IsDependency: &model.IsDependencyInputSpec{
-				DependencyType: model.DependencyTypeUnknown,
+				DependencyType: model.DependencyTypeDirect,
 				VersionRange:   "2.13.4.Final",
-				Justification:  isDepJustifyTopPkgJustification,
+				Justification:  isCDXDepJustifyDependsJustification,
 			},
 		},
 		{
 			Pkg:             cdxResteasyPack,
 			DepPkg:          cdxReactiveCommonPack,
 			DepPkgMatchFlag: model.MatchFlags{Pkg: model.PkgMatchTypeSpecificVersion},
 			IsDependency: &model.IsDependencyInputSpec{
-				DependencyType: model.DependencyTypeUnknown,
+				DependencyType: model.DependencyTypeIndirect,
```

Comment:
so I made a mistake in my explanation in the issue. This should also be marked as `DependencyTypeDirect` as `cdxResteasyPack` is a direct dependency to `cdxReactiveCommonPack` as you can see in the SBOM:
```
 "dependencies" : [
    {
      "ref" : "pkg:maven/org.acme/getting-started@1.0.0-SNAPSHOT?type=jar",
      "dependsOn" : [
        "pkg:maven/io.quarkus/quarkus-resteasy-reactive@2.13.4.Final?type=jar"
      ]
    },
    {
      "ref" : "pkg:maven/io.quarkus/quarkus-resteasy-reactive@2.13.4.Final?type=jar",
      "dependsOn" : [
        "pkg:maven/io.quarkus/quarkus-resteasy-reactive-common@2.13.4.Final?type=jar"
      ]
    }
 ```